MH-12919, REST Docs Dependencies

lkiesow opened the pull request:
This patch removes the inclusion of a lot of unnecessary resources
originating from a time where this module did hold more than the REST
documentation. All of the few necessary resources are now included.

This patch also includes an update for the now unsupported version 1.x
of jQuery, updated to v3.3.1.

Finally, this separates JavaScript and CSS to make using some security
features like Content-Security-Policy headers possible.

lkiesow committed:
MH-12919, REST Docs Dependencies

This patch removes the inclusion of a lot of unnecessary resources
originating from a time where this module did hold more than the REST
documentation. All of the few necessary resources are now included.

This patch also includes an update for the now unsupported version 1.x
of jQuery, updated to v3.3.1.

Finally, this separates JavaScript and CSS to make using some security
features like Content-Security-Policy headers possible.

staubesv commented on modules/runtime-info-ui-ng/src/main/resources/ui/js/jquery/jquery.js:
```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.3.1 | (c) JS Foundation and other contributors | jquery.org/license */
```
Should be included in NOTICES, right?

staubesv reviewed (changes requested):
Hi @lkiesow Besides jQuery probably should be mentioned in NOTICES, the PR is fine. Can you add the missing lines to NOTICES?

lkiesow committed:
MH-12919, Fix Non-Admin Runtime UI

This patch fixes the runtime interface on non-admin nodes which did not
include all necessary parts of the runtime-info-ui package.

lkiesow commented:
Makes sense. I fixed that.
I also fixed the runtime-info-ui package which had parts of the necessary resources included…
